events/events-router.js

```diff
@@ -15,16 +15,10 @@ router.get('/users/:id', validEventID, (req, res, next) => {
         Users.getInvited(id)
     ])
     .then(([organizedEvents, guestEvents]) => {
-        res.status(200).json({organizedEvents, guestEvents})
-    })
-    .catch(next)
-})
-//GET /events/:id/food
-router.get('/:id/food', (req, res, next) => {
-    const id = req.params.id
-    Events.getFoodList(id)
-    .then(list => {
-        res.status(200).json(list)
+        Events.getFoodList(id)
```

Comment:
I would recommend refactoring this so all of your queries are running at once. 

```
Promise.all([
        Events.getUserID(id),
        Users.getInvited(id),
        Events.getFoodList(id)
    ])
    .then(([organizedEvents, guestEvents, list]) => {
            res.status(200).json(({organizedEvents, guestEvents, menu: list}))
        })
    })
```

Reply:
oh 🙈 Yeah that would be a better flow. Thank you! I forgot that I could do that too haha